ajax/libs/labelauty/1.1.0/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "jquery-labelauty",
+  "version": "1.1.0",
+  "homepage": "https://github.com/fntneves/jquery-labelauty",
+  "authors": [
+    "Francisco Neves <contact@francisconeves.com>"
+  ],
+  "repository": {
+    "type":"git",
+    "url":"git://github.com/fntneves/jquery-labelauty"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/fntneves/jquery-labelauty"
+  },
```

Comment:
Please ａｌｓｏ add fileMap. You can take a look at https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md.